### DISCUSSION TITLE
PRODUCTION CERTIFICATES HELP - CERTIFICATES PERIOD!  ;O)

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6070] [Q&A]
Hi, I have not seen any updates on how to properly set up certificates for production environment.   I have a CRT, RSA Private Key, and a CABUNDLE. I imported these from LetsEncrypt. Each one of these have a bunch of random letters which I am not sure how to proceed with them or where to use them.  Would someone with more experience on this, let me know what I need to do with these certs for MeshCentral to make it more secure? Right now, I have an "X Not Secure" in my browser.  How would I fix that also? I appreciate any help!  Thanks for your time!

**Comment by silversword411** [comment]
See my description if you need paid support.
